packages/cfd/src/Containers/mt5-desktop-download-option.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import './mt5-mobile-download-option.scss';
+import { Icon, Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { getDownloadOptions, getPlatformMt5DownloadLink } from '../Helpers/constants';
+import { CFD_PLATFORMS, getCFDPlatformLabel, getPlatformSettings, getUrlBase } from '@deriv/shared';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+
+type TMT5MobileDownloadOptionProps = {
+    account_title: string;
+    mt5_trade_account: DetailsOfEachMT5Loginid;
+};
+const MT5MobileDownloadOption = ({ account_title, mt5_trade_account }: TMT5MobileDownloadOptionProps) => {
+    const desktopDownloadOptions = getDownloadOptions({ mt5_trade_account }).filter(
+        option => option.device === 'desktop'
+    );
+
+    return (
+        <>
```

Comment:
React.Fragment. We are already using Fragment and we'd better do the same for the newly added code as well.